src/main/webapp/aboutus.html

```diff
@@ -51,8 +51,8 @@ <h1>About Our Team</h1>
 
     <div class="row mt-3">
       <div class="col-sm">
-        <div class="card" style="width: 38rem;">
```

Comment:
So, simply removing width does the trick. Awesome.

Reply:
Yes, because "rem" is an absolute measurement. And it is put on top of the card class a, however, we want a full-width card.